getting-started/setup-building.rst

```diff
@@ -483,31 +483,55 @@ Regenerate ``configure``
 
 If a change is made to Python which relies on some POSIX system-specific
 functionality (such as using a new system call), it is necessary to update the
-``configure`` script to test for availability of the functionality.
+:cpy-file:`configure` script to test for availability of the functionality.
+Python's :file:`configure` script is generated from :cpy-file:`configure.ac`
+using GNU Autoconf.
 
-Python's ``configure`` script is generated from ``configure.ac`` using Autoconf.
-Instead of editing ``configure``, edit ``configure.ac`` and then run
-``autoreconf`` to regenerate ``configure`` and a number of other files (such as
-``pyconfig.h``).
+After editing :file:`configure.ac`, run ``make regen-configure`` to generate
+:file:`configure`, :cpy-file:`pyconfig.h.in`, and :cpy-file:`aclocal.m4`.
+When submitting a patch with changes made to :file:`configure.ac`,
+you must also include the changes in these generated files.
 
-When submitting a patch with changes made to ``configure.ac``, you should also
-include the generated files.
+The recommended and by far the easiest way to regenerate :file:`configure` is:
 
-Note that running ``autoreconf`` is not the same as running ``autoconf``. For
-example, ``autoconf`` by itself will not regenerate ``pyconfig.h.in``.
-``autoreconf`` runs ``autoconf`` and a number of other tools repeatedly as is
-appropriate.
+.. code-block:: bash
```

Comment:
This code block (like the others in this file) is `console`, not `bash`, as it includes the `$` prompt. Therefore, you should either change this:

```suggestion
.. code-block:: console
```

Or, like the other code blocks in this file, just use the default language, explicitly set to `console` above, via `::` instead of `.. code-block:: language` (also needs the suggestion to add that):

```suggestion
```